Automattic/wp-calypso
# i18n: Fix translations on the Use my domain screen

#### Changes proposed in this Pull Request

* Adds dynamic getters to the translated properties of `optionInfo` objects so that `__` is called every time they are accessed, to make sure the text is refreshed when translations are loaded.

#### Testing instructions
1. Change your user language to a non-English language.
1. Go to https://wordpress.com/domains/add/use-my-domain/{site-URL}
1. Enter a test domain name, such as example.com.
1. Make sure the mapping options are translated:
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/23708351/137494796-a0d5017a-66e5-4c32-8119-b35557c19d89.png">






Related to #325-gh-Automattic/i18n-issues
